dns/UnboundBL/+POST_INSTALL

```diff
@@ -0,0 +1,2 @@
+chmod 755 /usr/local/opnsense/scripts/OPNsense/UnboundBL/UnboundBL.sh
+chmod 755 /usr/local/opnsense/scripts/OPNsense/UnboundBL/data.sh
```

Comment:
This can be done by @fichtner after merge, no need for +POST_INSTALL. Just remove this file, for dev just keep in mind to do a chmod. I for myself always create a small script cd go into git dir, git pull && make upgrade and then do the chmod.

Reply:
@mimugmail is right - just check it into git with the right permission and you won't need this hack.